app/main.py

```diff
@@ -1,6 +1,37 @@
+import requests
+from dotenv import load_dotenv
+import os
+
+
+load_dotenv()
+URL = "http://api.weatherapi.com/v1/current.json"
+FILTER = "Paris"
+API_KEY = os.getenv("API_KEY")
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    payload = {
+        "key": API_KEY,
+        "q": FILTER,
+    }
+
+    response = requests.get(URL, params=payload)
+
+    if response.status_code != 200:
+        print(f"No data: error code {response.status_code}")
```

Comment:
Return in case there is not 200 response, further code should not execute after that.